docs/admin-invitations/0.1/README.md

```diff
@@ -13,151 +10,204 @@ a new DID to be provisioned for the invitation.
 - This version lacks paging of invitation lists.
 - This version lacks a way to filter invitations when listing.
 - The first example for each protocol message is for general usage; the second example specific to the ACA-Py Plugin Toolbox
-
+- Invitation create method will be changed slightly; the user will be able to specify whether to use Out-Of-Band or Connections protocol
 ### Protocol Messages
 - [create](#create)
+- [oob-create](#oob-create)
 - [invitation](#invitation)
+- [oob-invitation](#oob-invitation)
 - [invitation-get-list](#invitation-get-list)
 - [invitation-list](#invitation-list)
-
+Toolbox Plugin agents are able to send and receive either [Connections](https://github.com/hyperledger/aries-rfcs/blob/main/features/0160-connection-protocol/README.md#1-connection-request) Protocol or [Out-Of-Band](https://github.com/hyperledger/aries-rfcs/tree/main/features/0434-outofband) Protocol invitations, through the Admin Protocol messages. Agents receiving Connections Protocol invitations will use Connections Protocol to connect to the sender. Agents receiving OOB Protocol invitations will be able to choose between using Connections or [DID Exchange](https://github.com/hyperledger/aries-rfcs/tree/main/features/0023-did-exchange) protocols to respond and connect to the sender, which will be listed in the invitation's [handshake protocols](#out-of-band-specific-attributes). 
```

Comment:
While this is true (and a valuable description of current functionality in the plugin) this isn't the right doc for this. These docs define the protocol which is by definition not specific to any particular implementation. Another agent (one not based on the ACA-Py toolbox plugin) could implement the same protocol. In that case, this aside would, unfortunately, be completely irrelevant.